index.js

```diff
@@ -5,19 +5,54 @@ const Messenger = require('./src/Messenger');
 const constants = require('./src/constants');
 require('dotenv').config();
 const api = require('./src/api');
-const Botkit = require('botkit');
 const server = require('./server.js');
 const helpers = require('./helpers');
+const cluster = require('cluster');
```

Comment:
I introduced node clusters to enhance the performance of the app. It optimizes the use of the available CPUs in the machine. It also makes the bot more available by serving multiple instances of the bot in the various workers.

For more information see [How to Create a Node.js Cluster for Speeding Up Your Apps](https://www.sitepoint.com/how-to-create-a-node-js-cluster-for-speeding-up-your-apps/) and [Scaling-out with Node Clusters](https://medium.com/the-andela-way/scaling-out-with-node-clusters-1dca4a39a2a) (An Andela Way article).